spec/views/organisations/form.html.erb_spec.rb

```diff
@@ -37,19 +37,20 @@
   end
 
   it 'should have categories in scroll box ordered by type and name' do
-    expect(rendered).to have_xpath("//div/strong/em[text()='What you do']/../../following-sibling::div[1]/label[text()='alligator']")
-    expect(rendered).to have_xpath("//div/strong/em[text()='What you do']/../../following-sibling::div[2]/label[text()='capybara']")
-    expect(rendered).to have_xpath("//div/strong/em[text()='Who you help']/../../following-sibling::div[1]/label[text()='crocodile']")
-    expect(rendered).to have_xpath("//div/strong/em[text()='Who you help']/../../following-sibling::div[2]/label[text()='guinea pig']")
-    expect(rendered).to have_xpath("//div/strong/em[text()='How you help']/../../following-sibling::div[1]/label[text()='iguana']")
-    expect(rendered).to have_xpath("//div/strong/em[text()='How you help']/../../following-sibling::div[2]/label[text()='rabbit']")
+    expect(rendered).to have_xpath("//h5[contains(., 'What you do')]/following-sibling::div[1]/label[text()[contains(.,'alligator')]]")
```

Comment:
if the class is needed for styling that's fine - if it's not I'd add an id ... I'd try to avoid any mention of html elements where possible (e.g. h#, div, table etc.) and also complex sibling relations if practical.

I'm not sure what you mean by "Also I was wondering whether we could change the it statement to remove in scroll box, as it's not in a scroll box anymore."